aerogear/unifiedpush-admin-ui
# Aerogear 10204

## Motivation
https://issues.jboss.org/browse/AEROGEAR-10204

## What
Added page showing the list of the applications, installations and messages

## Testing
1. Start a local UPS with or without Keycloak authentication on port 9999
2. install all the webui deps with `npm install`
3. run the new webui with `npm run start`
4. point the browser at `http://localhost:3000`

If your UPS version is not the latest master, you will have to disable CORS checks.

If everything works, you should see a page with the list of all the UPS applications
